cmd/xl-v1-object.go

```diff
@@ -266,7 +267,7 @@ func (xl xlObjects) GetObject(bucket, object string, startOffset int64, length i
 
 		// Get the checksums of the current part.
 		checkSums := make([]string, len(onlineDisks))
-		var ckSumAlgo HashAlgo
+		var ckSumAlgo bitrot.Algorithm = 0xffffffff
 		for index, disk := range onlineDisks {
```

Comment:
It is an invalid value (because it much to high) Currently valid is 
 - 0: Poly1305
 - 1: GHASH
 - 2: BLAKE2b512
 - 3: SHA-256

Your right, that should be changed - Maybe making [unknownAlgorithm](https://github.com/aead/minio/blob/erasure/pkg/bitrot/bitrot.go#L27) visible? Then it should be clear...